atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/BatchColumnRangeSelection.java

```diff
@@ -40,8 +41,8 @@ public static BatchColumnRangeSelection create(ColumnRangeSelection columnRangeS
 
     @JsonCreator
     public static BatchColumnRangeSelection create(
-            @JsonProperty("startCol") byte[] startCol,
-            @JsonProperty("endCol") byte[] endCol,
+            @JsonProperty("startCol") byte @Nullable [] startCol,
```

Comment:
these nulls are only passed to column range selection, which allows nulls, see below